### ISSUE TITLE
About the number of trainable and total parameters

### ISSUE THREAD

**Issue by kaiw7** [haoyi-duan/DG-SCT#3]
Hi, could you share the number of trainable and total parameters of your model on AVE, AVS, and AVQA tasks? Many thanks.

**Comment by haoyi-duan** [comment]
Hi, we have provided the information on the number of trainable and total parameters for AVE task in Section 4.7 of our paper, hope you find this helpful.

**Comment by kaiw7** [comment]
> Hi, we have provided the information on the number of trainable and total parameters for AVE task in Section 4.7 of our paper, hope you find this helpful.

I can find the number of parameters in AVE task but I cannot find it in other tasks. Could you provide them? Many thanks.

**Comment by kaiw7** [comment]
Hi, could you share the number of (trainable and total) parameters of your models for AVS and AVQA task? Many thanks.

**Comment by haoyi-duan** [comment]
(trainable / total params)
AVE: 43.6M 461.3M

AVS: S4: 61.5M 594.8M MS3: 56.86M 602.9M

AVQA: 50.0M 520.2M

**Comment by kaiw7** [comment]
> (trainable / total params) AVE: 43.6M 461.3M
> 
> AVS: S4: 61.5M 594.8M MS3: 56.86M 602.9M
> 
> AVQA: 50.0M 520.2M

Many thanks for your sharing.

**Comment by kaiw7** [comment]
I have a another question about trainable parameters. "grounding_gen/models_grounding_gen/lavish_grounding_gen_best.pt" is used to initial relevant part of model weights. During training, these part of weights is frozen or trainable? I mean the weights of grounding part will be updated or not during the training?

**Comment by haoyi-duan** [comment]
Trainable.

**Comment by kaiw7** [comment]
> Trainable.

Many thanks for your quick reply. Could I know how many gpu card and memory you used for runing AVQA task?

**Comment by haoyi-duan** [comment]
One NVIDIA A100 GPU, it is mentioned in Appendix.

**Comment by kaiw7** [comment]
I have another question about the trainable parameters. ''

> One NVIDIA A100 GPU, it is mentioned in Appendix.

How long it takes for training using one gpu card?

**Comment by kaiw7** [comment]
> I have another question about the trainable parameters. ''
> 
> > One NVIDIA A100 GPU, it is mentioned in Appendix.
> 
> How long it takes for training using one gpu card?

Do you use A100 with 80GB? Because I found you set batch size as 8 for AVQA and that batch size will occupy aound 80gb?

**Comment by haoyi-duan** [comment]
Yes, A100 with 80GB, takes about 2 or 3 days for training.

**Comment by kaiw7** [comment]
Could you share the training logs for AVQA task including the validation accuracy per epoch?

**Comment by haoyi-duan** [comment]
Hello, may I inquire if you have any other questions? Please feel free to ask all your questions at once so that I can provide you with thorough answers to each of them. Thank you!

**Comment by kaiw7** [comment]
> Hello, may I inquire if you have any other questions? Please feel free to ask all your questions at once so that I can provide you with thorough answers to each of them. Thank you!

I would like to ask you about the hyperparameter list and training logs for AVQA task? And whether you pick the model with the minimal validaiton accuracy for testing? Many thanks.

**Comment by haoyi-duan** [comment]
I'm pleased to inform you that you should be able to retrieve the parameter list from [here](https://github.com/haoyi-duan/DG-SCT/blob/main/DG-SCT/AVQA/net_grd_avst/base_options.py). As for the training logs, unfortunately, we're unable to provide them for you, and we apologize for any inconvenience this may cause. However, you are welcome to download our checkpoints for testing. 

I'm not entirely clear on "picking the model with the minimal validation accuracy for testing", we pick the model with the maximum validation accuracy for testing.

**Comment by kaiw7** [comment]
> I'm pleased to inform you that you should be able to retrieve the parameter list from [here](https://github.com/haoyi-duan/DG-SCT/blob/main/DG-SCT/AVQA/net_grd_avst/base_options.py). As for the training logs, unfortunately, we're unable to provide them for you, and we apologize for any inconvenience this may cause. However, you are welcome to download our checkpoints for testing.
> 
> I'm not entirely clear on "picking the model with the minimal validation accuracy for testing", we pick the model with the maximum validation accuracy for testing.

Many thanks for your reply. I mean the maximal validation accuracy not minimal*. The last question is what is the maxiumal validation accuracy?

**Comment by haoyi-duan** [comment]
About 76%

**Comment by kaiw7** [comment]
> (trainable / total params) AVE: 43.6M 461.3M
> 
> AVS: S4: 61.5M 594.8M MS3: 56.86M 602.9M
> 
> AVQA: 50.0M 520.2M

**Comment by kaiw7** [comment]
> (trainable / total params) AVE: 43.6M 461.3M
> 
> AVS: S4: 61.5M 594.8M MS3: 56.86M 602.9M
> 
> AVQA: 50.0M 520.2M

Hi , I found a problem for computing the number of parameters so I reopen this issue. 

In AVQA structure, you use the frozen ViT equipped with prompt module as AVE task and also add the spatial-temporal grounding module on the top for AVQA structure. So, # trainable params of AVQA = # trainable params of AVE + # params of task-specific param? In that case, # total params of AVQA = # total params of AVE + # params of task-specific param; However, 50.0M (AVQA)-43.6M(AVE) is not equal to 520.2M(AVQA)-461.3M(AVE), where the difference should be the params of frozen ViT. 

 Many thanks.

**Comment by kaiw7** [comment]
> > (trainable / total params) AVE: 43.6M 461.3M
> > AVS: S4: 61.5M 594.8M MS3: 56.86M 602.9M
> > AVQA: 50.0M 520.2M
> 
> Hi , I found a problem for computing the number of parameters so I reopen this issue.
> 
> In AVQA structure, you use the frozen ViT equipped with prompt module as AVE task and also add the spatial-temporal grounding module on the top for AVQA structure. So, # trainable params of AVQA = # trainable params of AVE + # params of task-specific param? In that case, # total params of AVQA = # total params of AVE + # params of task-specific param; However, 50.0M (AVQA)-43.6M(AVE) is not equal to 520.2M(AVQA)-461.3M(AVE), where the difference should be the params of frozen ViT.
> 
> Many thanks.

Hi, could you give me more ideas about the question I mentioned above? Thank you very much.
